packages/strapi-plugin-i18n/controllers/content-types.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
       return ctx.badRequest('model.not.localized');
     }
 
-    let params = modelDef.kind === 'singleType' ? {} : { id };
+    let params = modelDef.kind === 'singleType' ? { _locale: 'all' } : { id };
```

Comment:
why not `_locale: locale` ?

Reply:
`locale` is the locale that doesn't exist yet and that the front would like to create.
The found entity is only used to get the `nonLocalizedAttributes`. With the CT we have the `id` of a related entity but not with the ST and any will do.

Reply:
Good idea, that works too